keyboards/mechmini/v2/keymaps/wsturgiss/keymap.c

```diff
@@ -0,0 +1,98 @@
+  //Tap once for , twice for -
+  [TD_COMM_MINUS]  = ACTION_TAP_DANCE_DOUBLE(KC_COMM, KC_MINUS),
+  //Tap once for . twice for =
+  [TD_DOT_EQUAL]  = ACTION_TAP_DANCE_DOUBLE(KC_DOT, KC_EQUAL),
+  //Tap once for / twice for '\'
+  [TD_SLASH_BACKSLASH]  = ACTION_TAP_DANCE_DOUBLE(KC_SLSH, KC_BSLS)
+};
+
+#define MODS_CTRL_MASK  (MOD_BIT(KC_LSHIFT)|MOD_BIT(KC_RSHIFT))
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[base] = LAYOUT_2u_space_ortho(
+		KC_TAB,     KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_BSPC,
+		KC_ESC,     KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_ENT,
+		KC_LSPO,    KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     TD(1),    TD(2),    TD(3),    KC_RSPC,
+		KC_LCTL,    _______,  KC_LALT,  KC_LGUI,  MO(1),        KC_SPC,         MO(2),    KC_VOLD,  KC_MPLY,  KC_VOLU,  KC_GRV),
+
+	[raise] = LAYOUT_2u_space_ortho(
+		_______,    KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_DEL,
+		_______,    _______,  _______,  _______,  _______,  _______,  KC_LEFT,  KC_DOWN,  KC_UP,    KC_RIGHT, KC_QUOT,  _______,
+		_______,    KC_HOME,  KC_END,   _______,  _______,  _______,  _______,  _______,  KC_LBRC,  KC_RBRC,  _______,  _______,  
+		_______,    _______,  _______,  _______,  _______,      _______,        _______,  KC_MPRV,  _______,  KC_MNXT,  RST),
+
+	[lower] = LAYOUT_2u_space_ortho(
+		_______,    KC_1,     KC_2,     KC_3,     KC_4,     KC_5,     KC_6,     KC_7,     KC_8,     KC_9,     KC_0,     _______,
+		_______,    RGB_TOG,  RGB_MOD,  RGB_VAI,  RGB_VAD,  _______,  _______,  KC_4,     KC_5,     KC_6,   KC_KP_PLUS, _______,
+		_______,    RGB_SAI,  RGB_SAD,  RGB_HUI,  RGB_HUD,  _______,  _______,  KC_1,     KC_2,     KC_3,  KC_KP_MINUS, _______,
+		_______,    TBL_FLP,  _______,  _______,  _______,      _______,        _______,  KC_0,     _______,  _______,  _______)
+
+
+};
+
+//change colors and rgb modes on layer change
+uint32_t layer_state_set_user(uint32_t state) {
+    switch (biton32(state)) {
+    case raise:
+        rgblight_mode(1);
+        rgblight_setrgb_purple ();
+        break;
+    case lower:
+        rgblight_mode(1);
+        rgblight_setrgb(0x00, 0xa3, 0x0d);
+        break;
+    default: // for any other layers, or the default layer
+        rgblight_mode(5);
+        rgblight_setrgb(0xFF, 0xB6, 0x00);
+        break;
+    }
+  return state;
+};
+
+//Macros
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case TBL_FLP:
+      if (record->event.pressed) {
+        // when keycode tableflip is pressed
+        set_unicode_input_mode(UC_OSX);
+        send_unicode_hex_string("0028 30CE 0CA0 75CA 0CA0 0029 30CE 5F61 253B 2501 253B");
+      } else {
+        // when keycode tableflip is released
+      }
+      break;
+    case RST:
```

Comment:
You don't need this case at all; QMK now has a core keycode `EEPROM_RESET` (aliased as `EEP_RST`) to reset the EEPROM.
https://github.com/qmk/qmk_firmware/blob/b0532c433edbe9419048882c1f14e0b54cd4e2b0/quantum/quantum.c#L323-L327
https://github.com/qmk/qmk_firmware/blob/b0532c433edbe9419048882c1f14e0b54cd4e2b0/quantum/quantum_keycodes.h#L610